Defra.Cdp.Backend.Api/Services/Secrets/PendingSecretsService.cs

```diff
@@ -70,7 +70,7 @@ CancellationToken cancellationToken
                     SecretKey = registerPendingSecret.SecretKey, Action = registerPendingSecret.Action
                 });
 
-        await Collection.UpdateOneAsync(filter, update, new UpdateOptions { IsUpsert = true }, cancellationToken);
+      _logger.LogInformation("Registering pending secret {SecretKey} for {Service} in {Environment}", registerPendingSecret.SecretKey, registerPendingSecret.Service, registerPendingSecret.Environment);
```

Comment:
is removing the updateOneAsync intentional?

Reply:
Nope, a bit of git failure. Will fix